### PR TITLE
Ria 3861 ooc refuse

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3861

### Change description ###
RIA-3861 OOC appeal - no sponsor-Refuse a HR or protection claim

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by divisathyan — APPROVED** [review]
LGTM

File: src/functionalTest/resources/scenarios/RIA-3861-out-of-country-non-sponsor-client-left-country.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "eventId": "startAppeal",
+      "state": "appealStarted",
+      "caseData": {
+        "template": "minimal-appeal-started.json",
+        "replacements": {
+          "appellantInUk": "No",
+          "hasCorrespondenceAddress": "Yes",
+          "appellantOutOfCountryAddress": "Kabul, Afghanistan",
+          "hasSponsor": "No",
+          "decisionLetterReceivedDate":"{$TODAY-7}",
```
Comment (mkayad) [inline src/functionalTest/resources/scenarios/RIA-3861-out-of-country-non-sponsor-client-left-country.json:17]: dateClientLeaveUk should be used in here, not decisionLetterReceivedDate

Comment (Lee365) [reply src/functionalTest/resources/scenarios/RIA-3861-out-of-country-non-sponsor-client-left-country.json]: Test corrected with dateClientLeaveUk

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Pull request merged**